### ISSUE TITLE
Problem with blending modes

### ISSUE THREAD

**Issue by Roskaiu** [nftchef/art-engine#162]
When I apply the merge mode it compiles correctly but doesn´t apply the blend mode.

    { name: "LaserEye", options: { blend: MODE.screen }

We will be very grateful

**Comment by Roskaiu** [comment]
Resolved.

/********************
I forgot to add the empty layer 
None#80
/********************

**Comment by Roskaiu** [comment]
.

**Comment by lucidreveur** [comment]
I am having similar issues with incorrect blend modes. Encountered an issue with the blend mode using the generator on an M1 Mac. Overlay does not have the same results as it is in photoshop / CanvasRenderingContext2D.globalCompositeOperation. My code: { name: "layer", blend: 'overlay' }. I checked the layer files and rebuilt them in photoshop. The overlay looks very dark. I took the same exact files in the previous generator on my old NON-M1 machine and they looked fine. I spent a long time trying to figure it out reading Github and on the readme. I read the // see src/blendMode.js for available blend mode. It was also recommended to find other people with similar issues of other people using node canvas.
